### PR TITLE
Created plot for MP2

### PR DESCRIPTION
**Pull request by heschmidt**

### CODE REVIEW

File: 390/HALEY_SCHMIDT.Rmd [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  filter(dbh != "NULL") %>%
+  mutate(dbh = as.numeric(dbh)) %>%
+  select(sp, CensusID, dbh) %>%
+  group_by(sp) %>% 
+  summarize(avg_dbh_sp = mean(dbh),
+            census_year = mean(CensusID))
+
+stem_joined <- rbind(stem1, stem2, stem3)
+
+genus_dbh <- stem_joined %>%
```
Comment (rudeboybert) [inline 390/HALEY_SCHMIDT.Rmd]: Hey Haley, here is a code review! A heads up that `scbi_spptable` is missing. You'll need to load that to ensure the .Rmd is completely reproducible

Comment (rudeboybert) [reply 390/HALEY_SCHMIDT.Rmd]: Could you fix this and push the changes?

Comment (rudeboybert) [reply 390/HALEY_SCHMIDT.Rmd]: Did you remember to push the change? b/c the code that loads `scbi_spptable` isn't there, and hence the file won't knit. It's a good idea to always knit once before pushing changes to make sure things look good.

Comment (heschmidt) [reply 390/HALEY_SCHMIDT.Rmd]: I tried again but I'm not sure if it worked, maybe we can look at this after class?

Comment (rudeboybert) [reply 390/HALEY_SCHMIDT.Rmd]: I was able to knit your file, so it worked! One way you can check if your pushes were made is to look at the "Files Changed" tab on the top right of this page (The tabs are Conversation, Commits, Checks, Files Changes). That shows the "diff" = difference between what's in `master` vs your `HaleyMP2` branch

**Comment by rudeboybert** [comment]
Hmm, your icons are still coming out grey. let's discuss after class. Please have that Happy Git R link I sent in Slack open.

Otherwise, you're done! I'm going to merge your branch into `master`

**Pull request merged**